data/data-pipeline/data_pipeline/tests/score/test_output.py

```diff
@@ -365,8 +384,16 @@ def test_imputed_tracts(final_score_df):
     )
 
     # Make sure that no tracts with population have null imputed income
+    # We DO NOT impute income 
```

Comment:
nitty nit: maybe just finish the comment with `We DO NOT impute income for island areas.`

Reply:
Fixed in b741001be6e1918273ddbc77f4431d507011d5df